docs/references/api.rst

```diff
@@ -613,6 +629,544 @@ like ``systemctl start ubuntu-advantage.service``.
+
+        - ``uaclient.api.u.pro.security.fix import NoOpData``
+
+          .. list-table::
+             :header-rows: 1
+
+             * - Field Name
+               - Type
+               - Description
+             * - ``status``
+               - *str*
+               - The status of the CVE when no operation can be performed
+
+        - ``uaclient.api.u.pro.security.fix import PackageCannotBeInstalledData``
+
+          .. list-table::
+             :header-rows: 1
+
+             * - Field Name
+               - Type
+               - Description
+             * - ``binary_package``
+               - *str*
+               - The binary package that cannot be installed
+             * - ``binary_package_version``
+               - *str*
+               - The version of the binary package that cannot be installed
+             * - ``source_package``
+               - *str*
+               - The source package associated with the binary package
+
+        - ``uaclient.api.u.pro.security.fix import SecurityIssueNotFixedData``
+
+          .. list-table::
+             :header-rows: 1
+
+             * - Field Name
+               - Type
+               - Description
+             * - ``source_packages``
+               - *List[str]*
+               - A list of source packages that cannot be fixed at the moment
+             * - ``status``
+               - *str*
+               - The status of the CVE regarding those packages
+
+      - Raised exceptions:
+
+        - No exceptions raised by this endpoint.   
+
+   .. tab-item:: CLI interaction
+      :sync: CLI
+
+      - Calling from the CLI:
+
+        .. code-block:: bash
+
+           pro api u.pro.security.fix.cve.plan.v1 --data '{"cves": [...]}'
+
+      - Expected attributes in JSON structure:
+
+        .. code-block:: json
+
+           {
+              "cves_data": {
+                  "expected_status": "fixed",
+                  "cves": [
+                    {
+                        "title": "CVE-1234-56789",
+                        "expected_status": "fixed",
+                        "plan": [
+                            {
+                                "operation": "apt-upgrade",
+                                "order": 1,
+                                "data": {
+                                    "binary_packages": ["pkg1"],
+                                    "source_packages": ["pkg1"],
+                                }
+                            }
+                        ],
+                        "warnings": [],
+                        "error": null
+                    }
+                  ]
+              }
+           }
+
+u.pro.security.fix.usn.plan.v1
+===============================
+
+This endpoint shows the necessary steps required to fix USNs in the system without
+executing any of those steps.
+
+- Introduced in Ubuntu Pro Client Version: ``29~``
+- Args:
+
+  - ``usns``: A list of USNs (i.e CVE-6188-1) titles
```

Comment:
I'm not sure if this is intended or not, but a couple of times in this section about the usn endpoint there are references to CVE (like here). Might be worth a double-check to make sure that any references to CVE for this endpoint are intended (I'm not sure!)

Reply:
Good catch @s-makin, this is indeed incorrect. I will fix that